tests/serverpod_test_server/lib/src/endpoints/column_operations/column_duration.dart

```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: deprecated_member_use_from_same_package
```

Comment:
These are temporary, right?

Reply:
Not really temporary for this file but I think these tests should be deleted when we remove the deprecated calls. For now I think it still makes sense to run the tests for the old methods as we still support them!

Reply:
Cool, just wanted to understand. Makes sense that we are testing the deprecated methods obviously!

Reply:
Good point @SandPod I have copy pasted them for the new methods!